src/utils/topic.py

```
@@ -0,0 +1,199 @@
+    res = None
+
+    while res is None:
+        try:    
+            llm = ChatOpenAI(model="gpt-4o-mini", temperature=0.2)
+            ans = llm.invoke([SystemMessage(content=system), HumanMessage(content=user)]).content
+            res = json.loads(ans)
+
+        except json.JSONDecodeError:
+            pass
+
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)